src/test/java/com/wikia/webdriver/testcases/adstests/mobileadstests/TestSlotsMobile.java

```diff
@@ -23,7 +23,7 @@
   private static final String MOBILE_PREFOOTER = "MOBILE_PREFOOTER";
 
   @Test(
-      groups = {"TestAdSlotsMobile_001", "TestAdSlotsMobile"},
+      groups = {"MobileAds", "MercuryAds", "TestAdSlotsMobile_001", "TestAdSlotsMobile"},
```

Comment:
If you introduced the new `"MercuryAds"` group, please, add the group to all mercury tests: http://qa-s2.wikia-prod:8080/view/Ads%20Synthetic/job/ZZZads-mercury-mobile/